views/js/runner/plugins/content/accessibility/keyNavigation.js

```diff
@@ -448,14 +538,19 @@ define([
         init: function init() {
             var self = this;
             var testRunner = this.getTestRunner();
+            var testData = testRunner.getTestData() || {};
+            var testConfig = testData.config || {};
+            var plaginState = _.defaults((testConfig.plugins || {})[name] || {}, defaultPluginState);
```

Comment:
Typo in the name. Also, this is a config, not a state :)

Reply:
fixed